pkg/sources/github/github.go

```
@@ -372,7 +373,11 @@ func (s *Source) enumerate(ctx context.Context, reporter sources.UnitReporter) e
 	// Report any values that were already configured.
 	for _, name := range s.filteredRepoCache.Keys() {
 		url, _ := s.filteredRepoCache.Get(name)
-		_ = dedupeReporter.UnitOk(ctx, RepoUnit{name: name, url: url})
+		url, err := s.ensureRepoInfoCache(ctx, url)
+		if err != nil {
+			_ = dedupeReporter.UnitErr(ctx, err)
```

Why are we discarding these errors?

---

I was being lazy :(